### ISSUE TITLE
Puzzle is Impossible to Solve

### ISSUE THREAD

**Issue by ng-nick** [Bram-Hub/Legup#42]
Puzzle is currently impossible to solve, used "Touching tents" and "Too few tents" to show this

![image](https://user-images.githubusercontent.com/80063060/135740855-ff5dbf76-5e97-4e48-9edc-7dc6f68306aa.png)
![image](https://user-images.githubusercontent.com/80063060/135740864-8e486ca6-a808-4457-ba2f-4fe28985f60d.png)

Problem mainly arises from the tree in row E, which must have a tent beneath it in order to satisfy the number of tents per row rule. The rows in the top of the grid cannot satisfy the tents per row rule without contradicting the "touching tents" rule and vice versa.

**Comment by Chase-Grajeda** [comment]
Please attach file

**Comment by ng-nick** [comment]
Yes I can open that and also look for valid tree tent puzzles as well. On the topic of valid/invalid puzzles, I came across another puzzle today that is also impossible to solve. I am not sure how many are actually impossible to solve but this seems like a recurring theme for this particular puzzle.
